airbyte-integrations/connectors/source-auth0/metadata.yaml

```diff
@@ -8,7 +8,7 @@ data:
     cloud:
       enabled: true
   connectorBuildOptions:
-    baseImage: airbyte/airbyte-python-connectors-base:0.1.0
+    baseImage: airbyte/python-connector-base:1.0.0
```

Comment:
Looks like this is the change you're (@bnchrch) referring to here:

> I updated the python metadata template to include the correct base image. And found that source-auth0 connector has already defined the field. I selfishly used this opportunity to have a connector using the new base images. Theres an argument to be made for us to just remove that line from their metadata.yaml since the field isnt current being used on master LMK your thoughts.

I don't think I saw the change to the templates in this PR - why exactly did source-auth0 have this set before? Were the templates added before and this got created afterwards?

I think it's fine to update if it's not doing anything and we'll want to change it anyway.

Looking at this a question came up though, do we always have to specify the base image? If it's not set do we infer "oh this is a python connector so I'll use the latest base image" when building?

Reply:
> I don't think I saw the change to the templates in this PR - why exactly did source-auth0 have this set before? Were the templates added before and this got created afterwards?

Yes that seems to be the case. It looks like it was part of the PR that added the new metadata field

> Looking at this a question came up though, do we always have to specify the base image? If it's not set do we infer "oh this is a python connector so I'll use the latest base image" when building? Also, what exactly happens here when the base image is not set?
You always need to either 
1. Set the base image in the metadata.yaml (encouraged)
2. Or have a dockerfile (discouraged but useful for thirdparty)

If you have neither then the build will fail.